test/MutationDataTest.py

```diff
@@ -123,6 +123,13 @@ def testPickleable(self):
         import cPickle
         cPickle.dump(m, open("out/testMDPickle.pkl", 'w'))
 
+    def test_copy(self):
+        """Test annotation copy """
```

Comment:
Could you add some mutation operations to the copy annotation and show that the original is unchanged?  i.e. prove that it's a proper deep copy